src/loss.py

```diff
@@ -0,0 +1,52 @@
+"""
+Loss
+"""
+import torch
+import torch.nn as nn
+
+
+class OZELoss(nn.Module):
```

Comment:
This loss is misleading, as it is not the one used for the challenge (which is the usual MSE). I defined this new loss later based on the literature in building management optimization.

For this repo, I think it's better we remove it. As for the transformer repo, we should rename it and improve the documentation.

Reply:
Makes sense.